### PR TITLE
feat: add more message status icons

### PR DESCRIPTION
**Pull request by trOnk12**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

- Add support for read message status and delivered. 

Needs releases with:

(https://github.com/wireapp/kalium/pull/1943)

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

image

image

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [840](https://10.10.124.134/job/AR-build-pipeline/840/) **failed**.

**Review by borichellow — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [863](https://10.10.124.134/job/AR-build-pipeline/863/) **failed**.

**Comment by AndroidBob** [comment]
Build [917](https://10.10.124.134/job/AR-build-pipeline/917/) **failed**.

**Review by alexandreferris — APPROVED** [review]

File: app/src/main/kotlin/com/wire/android/mapper/MessageMapper.kt [hunk 1]
```diff
@@ -167,12 +167,16 @@ class MessageMapper @Inject constructor(
...
+            when (val status = message.status) {
+                Message.Status.Pending -> MessageFlowStatus.Sending
+                Message.Status.Sent -> MessageFlowStatus.Sent
+                is Message.Status.Read -> MessageFlowStatus.Read(status.readCount)
+                Message.Status.Failed -> MessageFlowStatus.Failure.Send.Locally(isMessageEdited)
+                Message.Status.FailedRemotely -> MessageFlowStatus.Failure.Send.Remotely(
+                    isEdited = isMessageEdited,
+                    backendWithFailure = message.conversationId.domain
+                )
+                Message.Status.Delivered -> MessageFlowStatus.Delivered
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/mapper/MessageMapper.kt:179]: Do we have tests for this mapper?

**Comment by AndroidBob** [comment]
Build [949](https://10.10.124.134/job/AR-build-pipeline/949/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7252082)(7252082)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7252082)(7252082)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/mapper/MessageMapper.kt (hunk 1 above)
Comment (trOnk12) [reply app/src/main/kotlin/com/wire/android/mapper/MessageMapper.kt:179]: added tests

**Comment by AndroidBob** [comment]
Build [986](https://10.10.124.134/job/AR-build-pipeline/986/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7261381)(7261381)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7261381)(7261381)-staging-compat.apk)

**Pull request closed without merging**